frontend/app/routes/_gcweb-app.update-phone-number.tsx

```diff
@@ -14,8 +14,21 @@ export async function loader({ request }: LoaderFunctionArgs) {
   return json({ userInfo });
 }
 
+function parsePhoneNumber(phoneNumber: string): string {
```

Comment:
Feel free to ignore, but this seems kind of clunky.  It should also go in a file under the `/utils` folder as well.
I feel a more 'all-encompassing' approach would be the following:

```javascript
function parsePhoneNumber(phoneNumber: string): string {
  let digits = phoneNumber.replace(/\D/g, '');
  return `(${digits.slice(0, 3)})${digits.slice(3, 6)}-${digits.slice(6)}`;
}
```

Reply:
Updated to use the libphonenumber library which uses formatNational() so a custom utils function is no longer needed.